include/sc2api/sc2_unit.h

```diff
@@ -9,6 +9,7 @@
 #include "sc2_typeenums.h"
 #include <vector>
 #include <unordered_map>
```

Comment:
Do we still need `#include <unordered_map>` after removal of `units_previous_map_`?

Reply:
yes, it is used elsewhere.